documentation/contributing/pullrequests/index.markdown

```diff
@@ -151,5 +151,5 @@ If each step ends with issues, they need to be fixed before moving on.
 ### Release versioning policy
 
 * We use the minor version to differentiate the releases from different development branches.
-  * As of December 2016, Indigo 0.7.x (indigo-devel branch), Jade 0.8.x (jade-devel), Kinetic 0.9.x (kinetic-devel).
+  * As of May 2019, Kinetic 0.9.x (kinetic-devel), Melodic 1.0.x (melodic-devel)
```

Comment:
@130s is this correct?

Reply:
That seems correct.
- [Kinetic](http://repositories.ros.org/status_page/kinetic_default.html?q=ros-kinetic-moveit) 0.9.15
- [Melodic](http://repositories.ros.org/status_page/melodic_default.html?q=ros-melodic-moveit) 1.0.1